rakelib/routes_csv.rake

```diff
@@ -0,0 +1,46 @@
+namespace :routes do
+  desc 'Print out all defined routes in CSV format.'
+  task :csv => :environment do
+
+    class CSVFormatter
+      def initialize
+        @buffer = []
+      end
+
+      def result
+        @buffer.join("\n")
+      end
+
+      def section_title(title)
+        @buffer << "\n#{title}:"
+      end
+
+      def section(routes)
+        routes.map do |r|
+          # @buffer << "#{r[:name]},#{r[:verb]},#{r[:path]},#{r[:reqs]}"
```

Comment:
We're okay having commented-out code in this file?

Reply:
Yes, It's just an example if you need this to return alternate info